area_detector_handlers/_xspress3.py

```diff
@@ -7,8 +7,7 @@
 
 
 class BulkXSPRESS(HandlerBase):
-    specs = {"XSP3_FLY"}
-    HANDLER_NAME = "XSP3_FLY"
+    specs = {'XSP3_FLY', 'XPS3_FLY', 'DEXELA_FLY_V1', 'MERLIN_FLY_STREAM_V1', 'MERLIN_FLY'} | HandlerBase.specs
```

Comment:
Is it worth writing down in code comments where each of these is known to be used? If you are tracking that somewhere else that the group can find it, that's fine too.

Reply:
That's a good idea. At the moment it's only mentioned in commit messages, if at all, but that's not a good way to track that.

Reply:
I'm also torn on the `| HandlerBase.specs`.

On the one hand I see the idea behind it, which is why I was tempted into adding it here to be consistent with the XSP3 handler below.

But thinking on it: HandlerBase.specs is `set()`, will likely always be `set()` and isn't even the real `HandlerBase` anyway, since we're faking it to avoid the databroker dependency. Arguably, it's also misleading, giving the impression spec inheritance is more nuanced than it actually is, and that `specs` has some functional role in it.

Any objections to removing this and other instances of `| HandlerBase.specs`?

Reply:
I have never liked it either, for the reasons you articulated well here. I think we can make an executive decision here to drop it.